specs-go/config.go

```diff
@@ -108,6 +108,14 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion specifies the operating system version, for example `10.0.10586`.
```

Comment:
(Historical - 10586 was a pre-release, IIRC Technical Preview 4 of Windows Server 2016)